core/client/ui/form-log-in.js

```diff
@@ -71,4 +71,5 @@ Template.formLogIn.helpers({
   email() { return Template.instance().email; },
   password() { return Template.instance().password; },
   loginMode() { return Template.instance().loginMode.get(); },
-});
+  contactURL() { const contactURL = Meteor.settings.public.permissions?.contactURL; return contactURL.length > 0 ? contactURL : undefined; }},  
+);
```

Comment:
weird style

Reply:
Is it even useful to check on the `contactURL` length ? As an empty string is evaluated to false in an Handlebars if.